### PR TITLE
Add output shape and output padding for Convolution Backprop SLTs.

### PR DESCRIPTION
**Pull request by sdurawa**
### Details:
 - add output_shape input and output_padding attribute into SLTs scope,
 - add SLT for Serialization.

### Tickets:
 - 46093

### CODE REVIEW

File: inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp [hunk 1]
```diff
@@ -139,6 +139,20 @@ std::shared_ptr<ngraph::Node> makeConvolutionBackpropData(const ngraph::Output<N
                                                           bool addBiases = false,
                                                           const std::vector<float> &biasesWeights = {});
 
+std::shared_ptr<ngraph::Node> makeConvolutionBackpropData(const ngraph::Output<Node> &in,
```
Comment (ilyachur) [inline inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp:142]: As I see we have 3 different builders for deconvolution. Do we need all these builders?

Comment (sdurawa) [reply inference-engine/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/builders.hpp:142]: Yes, we need another builder. The 2 existing builders are responsible for creating ConvolutionBackprop for this ctor [ctor](https://github.com/openvinotoolkit/openvino/blob/4bbbf66a69b339e27a22c18d9b2a485a22a77226/ngraph/core/include/ngraph/op/convolution.hpp#L137), and we need also builder for 2nd [ctor](https://github.com/openvinotoolkit/openvino/blob/4bbbf66a69b339e27a22c18d9b2a485a22a77226/ngraph/core/include/ngraph/op/convolution.hpp#L111) as it has 3rd input: output_shape.

**Comment by sdurawa** [comment]
@ggalieroc , @jdanieck , @ilyachur , please review. commit provides extended API for SLTs with output_padding attribute and output_shape input. Additionally old API is preserved as KMB and ARM plugins are still using it. It shall be removed (ticket 55568) when plugins will switch to new API.

**Review by jdanieck — APPROVED** [review]
Do we have a tickets for dependency removal in plugins? In comments I see only 55568 mentioned and it is not linked with other tickets.

**Comment by sdurawa** [comment]
@jdanieck, tickets to align SLTs for ARM and KMB plugins already exist: 56851 and 56852.

**Review by ilyachur — APPROVED** [review]

**Review by ggalieroc — APPROVED** [review]

**Comment by ilyachur** [comment]
@sdurawa Please update your branch to the latest master.

**Pull request merged**